src/accordion/accordion.directive.ts

```diff
@@ -0,0 +1,305 @@
+import {
+	AfterContentInit,
+	ChangeDetectorRef,
+	Component,
+	ContentChild,
+	ContentChildren,
+	Directive,
+	EventEmitter,
+	forwardRef,
+	Inject,
+	Input,
+	Output,
+	QueryList,
+	TemplateRef,
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+import { NgbAccordionConfig } from './accordion-config';
+import { NgTemplateOutlet } from '@angular/common';
+import { NgbCollapse } from '../collapse/collapse';
+
+let nextId = 0;
+
+@Component({
+	selector: '[ngbAccordionBody]',
+	standalone: true,
+	imports: [NgTemplateOutlet],
+	host: { '[class.accordion-body]': 'true' },
+	template: ` <ng-template [ngTemplateOutlet]="addToDOM() ? bodyTpl : null" ]></ng-template> `,
+})
+export class NgbAccordionBody {
+	constructor(@Inject(forwardRef(() => NgbAccordionItem)) private item: NgbAccordionItem) {}
+
+	@ContentChild(TemplateRef) bodyTpl: TemplateRef<any>;
+
+	addToDOM() {
+		return this.item.destroyOnHide === false || this.item.animatingBodyCollapse;
+	}
+}
+
+@Directive({
+	exportAs: 'ngbAccordionCollapse',
+	standalone: true,
+	selector: '[ngbAccordionCollapse]',
+	host: {
+		role: 'region',
+		'[class.accordion-collapse]': 'true',
+		'[id]': 'item.collapseId',
+		'[attr.aria-labelledby]': 'item.toggleId',
+	},
+	hostDirectives: [
+		{
+			directive: NgbCollapse,
+		},
+	],
+})
+export class NgbAccordionCollapse {
+	constructor(
+		@Inject(forwardRef(() => NgbAccordionItem)) public item: NgbAccordionItem,
+		public ngbCollapse: NgbCollapse,
+	) {}
+}
+
+@Directive({
+	selector: '[ngbAccordionToggle]',
+	standalone: true,
+	host: {
+		'[id]': 'item.toggleId',
+		'[disabled]': 'item.disabled',
+		'[class.accordion-button]': 'true',
+		'[class.collapsed]': 'item.collapsed',
+		'[attr.aria-controls]': 'item.collapseId',
+		'[attr.aria-expanded]': '!item.collapsed',
+		'[type]': '"button"',
+		'(click)': 'accordion.toggle(item.id)',
+	},
+})
+export class NgbAccordionToggle {
+	constructor(
+		@Inject(forwardRef(() => NgbAccordionItem)) public item: NgbAccordionItem,
+		@Inject(forwardRef(() => NgbAccordionDirective)) public accordion: NgbAccordionDirective,
+	) {}
+}
+
+@Directive({
+	selector: '[ngbAccordionHeader]',
+	standalone: true,
+	host: {
+		role: 'heading',
+		'[class.accordion-header]': 'true',
+	},
+})
+export class NgbAccordionHeader {
+	constructor() {}
+}
+
+@Directive({
+	selector: '[ngbAccordionItem]',
+	standalone: true,
+	host: {
+		'[class.accordion-item]': 'true',
+		'[id]': 'id',
+	},
+})
+export class NgbAccordionItem implements AfterContentInit {
```

Comment:
Must implement `OnDestroy` and unsubscribe, as you're leaking subscriptions (never unsubscribing)

Something like:
```ts
for (let s of this._subscriptions) {
  s?.unsubscribe();
}
```